SQF/dayz_code/stringtable.xml

```diff
@@ -14619,5 +14619,741 @@
+			<Czech>30 Meter Plot</Czech>
+		</Key>
+		<Key ID="STR_EPOCH_30METERPLOT_DESC">
+			<English>Plot Pole: A placeable pole that will allow you and your friends to build within a 30m radius. Ownership of plot pole is lost after owners death.</English>
+			<German>Plot Pole: A placeable pole that will allow you and your friends to build within a 30m radius. Ownership of plot pole is lost after owners death.</German>
+			<Russian>Plot Pole: A placeable pole that will allow you and your friends to build within a 30m radius. Ownership of plot pole is lost after owners death.</Russian>
+			<Spanish>Plot Pole: A placeable pole that will allow you and your friends to build within a 30m radius. Ownership of plot pole is lost after owners death.</Spanish>
+			<French>Plot Pole: A placeable pole that will allow you and your friends to build within a 30m radius. Ownership of plot pole is lost after owners death.</French>
+			<Czech>Plot Pole: A placeable pole that will allow you and your friends to build within a 30m radius. Ownership of plot pole is lost after owners death.</Czech>
+		</Key>
+		<Key ID="STR_EPOCH_METALPANEL">
+			<English>Metal Panel</English>
+			<German>Metal Panel</German>
+			<Russian>Metal Panel</Russian>
+			<Spanish>Metal Panel</Spanish>
+			<French>Metal Panel</French>
+			<Czech>Metal Panel</Czech>
+		</Key>
+		<Key ID="STR_EPOCH_METALPANEL_DESC">
+			<English>Metal Panel: Strong metal wall used for base defence.</English>
+			<German>Metal Panel: Strong metal wall used for base defence.</German>
+			<Russian>Metal Panel: Strong metal wall used for base defence.</Russian>
+			<Spanish>Metal Panel: Strong metal wall used for base defence.</Spanish>
+			<French>Metal Panel: Strong metal wall used for base defence.</French>
+			<Czech>Metal Panel: Strong metal wall used for base defence.</Czech>
+		</Key>
+		<Key ID="STR_EPOCH_DEERSTAND">
+			<English>Deer Stand</English>
+			<German>Deer Stand</German>
+			<Russian>Deer Stand</Russian>
+			<Spanish>Deer Stand</Spanish>
+			<French>Deer Stand</French>
+			<Czech>Deer Stand</Czech>
+		</Key>
+		<Key ID="STR_EPOCH_DEERSTAND_DESC">
+			<English>Deer Stand: can be used as a lookout.</English>
+			<German>Deer Stand: can be used as a lookout.</German>
+			<Russian>Deer Stand: can be used as a lookout.</Russian>
+			<Spanish>Deer Stand: can be used as a lookout.</Spanish>
+			<French>Deer Stand: can be used as a lookout.</French>
+			<Czech>Deer Stand: can be used as a lookout.</Czech>
+		</Key>
+		<Key ID="STR_EPOCH_LARGEDESERTCAMONET">
+			<English>Large Desert Camo Net</English>
+			<German>Large Desert Camo Net</German>
+			<Russian>Large Desert Camo Net</Russian>
+			<Spanish>Large Desert Camo Net</Spanish>
+			<French>Large Desert Camo Net</French>
+			<Czech>Large Desert Camo Net</Czech>
+		</Key>
+		<Key ID="STR_EPOCH_LARGEDESERTCAMONET_DESC">
+			<English>Large Camo Netting: A very large buildable object that can be used to cover objects.</English>
```

Comment:
STR_EPOCH_LARGEDESERTCAMONET_DESC and STR_EPOCH_LARGEFORESTCAMONET_DESC are duplicates. You can just use one for both.